xml/cl.xml

```diff
@@ -6100,5 +6107,20 @@ server's OpenCL/api-docs repository.
                 <command name="clGetKernelSuggestedLocalWorkSizeKHR"/>
             </require>
         </extension>
+        <extension name="cl_khr_integer_dot_product" supported="opencl">
+            <require>
+                <type name="CL/cl.h"/>
+            </require>
+            <require>
+                <type name="cl_device_integer_dot_product_capabilities_khr"/>
+            </require>
+            <require comment="cl_device_integer_dot_product_capabilities_khr">
+                <enum name="CL_DEVICE_INTEGER_DOT_PRODUCT_INPUT_4x8BIT_KHR"/>
+                <enum name="CL_DEVICE_INTEGER_DOT_PRODUCT_INPUT_4x8BIT_PACKED_KHR"/>
+            </require>
```

Comment:
The current header generation script will generate `#defines` for these enums in the order they are listed in the XML file.  The unpacked `4x8BIT_KHR` enum is bit 1 and the `4x8BIT_PACKED_KHR` enum is bit 0.  If we want bit zero to come before bit one I'll either need to update the script to sort the enum names based on bit number or we should switch the order here in the XML file.